### PR TITLE
added standal-ce-nginx configuration

### PR DESCRIPTION
**Pull request by realStandal**

### CODE REVIEW

File: .github/workflows/standal-ce-nginx.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  push:
+
+    # Only run on these branches
+    branches:
+      - main
+
+    paths:
+
+      # Implementation specific
+      - '.github/workflows/template.yml'
```
Comment (jeliasson) [inline .github/workflows/standal-ce-nginx.yml:13]: Change `template.yml` to `standal-ce-nginx.yml` :)

File: .github/workflows/standal-ce-nginx.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+name: 'standal-ce-nginx'
+
+on:
+  push:
+
+    # Only run on these branches
+    branches:
+      - main
+
```
Comment (jeliasson) [inline .github/workflows/standal-ce-nginx.yml:9]: Maybe add branch `standal-ce-nginx` to target builds before merging with main.

File: .github/workflows/standal-ce-nginx.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    paths:
+
+      # Implementation specific
+      - '.github/workflows/template.yml'
+      - 'docker/template/**'
```
Comment (jeliasson) [inline .github/workflows/standal-ce-nginx.yml:14]: This should be `- 'docker/standal-ce-nginx/**'`. Sorry, will make this more clear in the template CI pipe :)

**Review by jeliasson — APPROVED** [review]
LGTM. Have a look on the comments and update at a later stage ✌️

File: .github/workflows/standal-ce-nginx.yml (hunk 3 above)
Comment (jeliasson) [reply .github/workflows/standal-ce-nginx.yml:14]: Oh yeah, we need this change to trigger the workflow.

Comment (realStandal) [reply .github/workflows/standal-ce-nginx.yml:14]: No I think you're right, so only changes to this one trigger this one.

I completely glossed over every one of the "implementation specific" items 😬

**Pull request merged**